### PR TITLE
Extract ArchivesSpaceClient and ArchivistsToolkitClient into new module

### PR DESCRIPTION
**Pull request by mistydemeo**
The new Python module currently lives at [artefactual-labs/agentarchives](https://github.com/artefactual-labs/agentarchives).

(The fact that I keep mixing up "agentarchives" vs "archivesagent" maybe isn't a good sign, hopefully I default to the correct one soon!)

### CODE REVIEW

File: src/archivematicaCommon/requirements/base.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ bagit==1.5.2
 Django>=1.7,<1.8
 elasticsearch>=1.0.0,<2.0.0
 requests==2.7.0
-django-mysqlpool
\ No newline at end of file
+django-mysqlpool
+agentarchives
```
Comment (Hwesta) [inline src/archivematicaCommon/requirements/base.txt]: Is this in PyPI yet? Does it need to be for this to work?

Comment (mistydemeo) [inline src/archivematicaCommon/requirements/base.txt]: It'll be going into PyPI for the 1.6 release.

**Comment by Hwesta** [comment]
Remove the tests and fixtures too?

**Comment by mistydemeo** [comment]
Good catch, I'll remove those too.

**Comment by Hwesta** [comment]
:+1:

**Comment by mistydemeo** [comment]
Updated import paths.

Do we want pip to point at a git repo until this is published on pypi, or can we cope with it not being installable via pip in the interim?

**Comment by Hwesta** [comment]
Probably the git repo, so installs of qa/1.x still work.  Once they get published, we can update the requirements.txt

**Comment by mistydemeo** [comment]
Updated to use the git repo.

**Pull request merged**